war/pom.xml

```diff
@@ -96,6 +96,11 @@ THE SOFTWARE.
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.main</groupId>
+      <artifactId>websocket-jetty10</artifactId>
+      <version>${project.version}</version>
+    </dependency>
     <dependency>
       <groupId>org.jenkins-ci.main</groupId>
       <artifactId>websocket-jetty9</artifactId>
```

Comment:
It feels wrong to me to just add jetty 10 and not remove jetty 9.

Reply:
> It feels wrong to me

@vlatombe Whether it feels wrong to you or not (and I would suggest that you make requests based on explicit and objective reasoning rather than feelings), implicitly requesting that I remove Jetty 10 from `war/pom.xml` as you are doing here is not a request that can be accommodated (regardless of its underlying reasoning), because that would introduce test failures in any `JenkinsRule`-based test that uses WebSockets, such as `jenkins.agents.WebSocketAgentsTest` (because `JenkinsRule` still uses Jetty 9 and cannot be migrated to Java 11 and Jetty 10 until we drop support for older versions of Jenkins, even though that eventual migration has been tested and validated in https://github.com/jenkinsci/jenkins-test-harness/pull/453, https://github.com/jenkinsci/jenkins/pull/6802, and https://github.com/jenkinsci/bom/pull/1255). I suggest that you refrain from making implicit requests for change that would introduce test failures.

Reply:
I had missed the fact that `JenkinsRule` still relied on jetty 9. With that additional context, it looks perfectly fine.